core/inspector.rs

```diff
@@ -365,6 +365,9 @@ impl JsRuntimeInspector {
             w.poll_state = PollState::Parked;
             w.parked_thread.replace(thread::current());
           }
+          PollState::Parked => {
+            w.parked_thread.replace(thread::current());
+          }
```

Comment:
Why was this change needed?

Reply:
I meant to inquire if that was correct but forgot.. There were some inspector test failures on https://github.com/denoland/deno/pull/19463/commits/fc0f65a48307c2632fa23669be93f60f94caabdd because that variant was being hit now for some reason. That fixed it. But I don't know what changed there.

Reply:
Huh, interesting! I would be interested in fixing this for sure, any chance you could try to create a test that would hit this code path? Inspector is quite brittle and esoteric bit of code so I would encourage a lot of caution when changing its logic.

Reply:
> any chance you could try to create a test that would hit this code path?

It's hit by the test failures on https://github.com/denoland/deno/actions/runs/5242046717/jobs/9464946684, caused by something in the rest of this PR.